### PR TITLE
Align palette - Refactoring & Unit tests

### PR DESCRIPTION
**Pull request by TurtlePriest**

### CODE REVIEW

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java [hunk 1]
```diff
@@ -48,12 +36,16 @@
...
+    ResourceBundleUtil labels = ResourceBundleUtil.getBundle("org.jhotdraw.samples.svg.Labels");
+    JPanel p = new JPanel();
 
     /** Creates new instance. */
     public AlignToolBar() {
-        ResourceBundleUtil labels = ResourceBundleUtil.getBundle("org.jhotdraw.samples.svg.Labels");
         setName(labels.getString(getID() + ".toolbar"));
     }
+    public JPanel getJPanel(){
+        return p;
```
Comment (do-nat) [inline JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:47]: is this method called somewehere?

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java [hunk 2]
```diff
@@ -48,12 +36,16 @@
 public class AlignToolBar extends AbstractToolBar {
 
     private SelectionComponentDisplayer displayer;
+    ResourceBundleUtil labels = ResourceBundleUtil.getBundle("org.jhotdraw.samples.svg.Labels");
```
Comment (do-nat) [inline JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:39]: labels are only used for initialization, maybe undo to the move to a field but leave it in the constructor

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java [hunk 3]
```diff
@@ -69,75 +61,93 @@ public void setEditor(DrawingEditor newValue) {
...
+    
+    public void createBtnEast (JPanel p, GridBagConstraints gbc){
+        
+        AbstractButton btn;
+        
+        gbc.insets = new Insets(0, 3, 0, 0);
+        btn = new JButton(new AlignAction.East(editor, labels));
+        createBtn(btn);
+        p.add(btn, gbc);
+    }
```
[the remark this answers is not in the document]
Comment (do-nat) [reply JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:83]: why are these methods public? It was protected before, why did you make it public?

**Review by do-nat — CHANGES_REQUESTED** [review]

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java (hunk 1 above)
Comment (TurtlePriest) [reply JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:47]: Yes! I use it for testing ;)

File: JHotDraw/src/test/java/TestSuites/AlignQueueTestSuite.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ *
+ * @author menta
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({org.jhotdraw.draw.action.AlignActionTest.class, org.jhotdraw.samples.svg.gui.AlignToolBoxTest.class, org.jhotdraw.draw.action.AlignJGivenTest.class})
+public class AlignQueueTestSuite {
```
[the remark this answers is not in the document]
Comment (TurtlePriest) [reply JHotDraw/src/test/java/TestSuites/AlignQueueTestSuite.java:13]: Yeah, it's just a suite that runs all my test files. Not necessary at all though :P
Should i just remove it?

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (TurtlePriest) [reply JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:83]: Yeah, i looked into doing that, but wouldn't it become a large and messy method?

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java (hunk 1 above)
Comment (TurtlePriest) [reply JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:47]: I just realized i could make it protected, and acess it directly. I'll be removing that method

File: JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java (hunk 2 above)
Comment (TurtlePriest) [reply JHotDraw/src/main/java/org/jhotdraw/samples/svg/gui/AlignToolBar.java:39]: Not entirely sure what you mean here

**Comment by TurtlePriest** [comment]
I think i've resolved all your requests, but the consolidation of the methods

**Review by xmimmer — APPROVED** [review]
Looks fine after the refactorings you made suggested by donat.

**Review by do-nat — APPROVED** [review]
Looks better 🙃

**Pull request merged**